configs/raspberrypi3_wpe_ml_defconfig

```diff
@@ -85,6 +85,7 @@ BR2_PACKAGE_WPEFRAMEWORK_UX=y
 BR2_PACKAGE_WPEFRAMEWORK_WEBKITBROWSER_THREADEDPAINTING="2"
 BR2_PACKAGE_WPEFRAMEWORK_WEBSERVER=y
 BR2_PACKAGE_WPEFRAMEWORK_WIFICONTROL=y
+BR2_PACKAGE_WPEFRAMEWORK_NETFLIX=y
```

Comment:
I'd opt for leaving it out of the default wpe_ml_defconfig for now. If people need NF5 they can always select it themselves from menuconfig. Since there is still a lot on going I do not want to destabilize the default build.

Reply:
Thank Wouter for the comments, I will update it